Missing 'promise' dependency

This is against the version1 branch -

```
yo vvv:create
module.js:341
    throw err;
    ^

Error: Cannot find module 'promise'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/Users/watson/Development/tools/generator-vvv/lib/util/rc-config.js:4:16)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

--- a/lib/util/rc-config.js
+++ b/lib/util/rc-config.js
@@ -1,15 +1,15 @@
 'use strict';
 
 var rcFetch = require( 'rc' );
-var NPromise = require( 'promise' );
-var writeFile = NPromise.denodeify( require( 'fs' ).writeFile );
+var Promise = require( 'bluebird' );
+var writeFile = Promise.promisify( require( 'fs' ).writeFile );
 var join = require( 'path' ).join;
 var home = ( process.platform === "win32" ) ? process.env.USERPROFILE : process.env.HOME;
 
 /**
  * Factory method to create a new RC object and return it.
  *
- * @param  {String} appname The name of the new storage 
+ * @param  {String} appname The name of the new storage
  * @param  {String} level   Whether to store globally (per user) or locally.
  * @return {RC}             The constructed RC object.
  */
@@ -21,7 +21,7 @@ module.exports = function( appname, defaults ) {
  * Storage instances handle a json rc file saving.
  *
  * @constructor
- * @param {String} appname The name of the new storage 
+ * @param {String} appname The name of the new storage
  * @param {String} level   Whether to store globally (per user) or locally.
  */
 function RC( appname, defaults ) {

--- a/lib/util/recursive-find.js
+++ b/lib/util/recursive-find.js
@@ -1,8 +1,8 @@
 var path = require( 'path' );
 var fs = require( 'fs' );
-var NPromise = require( 'bluebird' );
-var readdir = NPromise.denodeify( fs.readdir );
-var stat = NPromise.denodeify( fs.stat );
+var Promise = require( 'bluebird' );
+var readdir = Promise.promisify( fs.readdir );
+var stat = Promise.promisify( fs.stat );
 
 module.exports = walkDir;
 
@@ -33,6 +33,6 @@ function readFiles( name, folder, processor, depth ) {
 				promises.push( statFile( name, fullPath, processor, depth ) );
 			}
 		}
-		return NPromise.all( promises );
+		return Promise.all( promises );
 	};
 }

